webapp/src/components/DatasetWarnings/DatasetDistribution.tsx

```diff
@@ -4,6 +4,22 @@ import MetricsDataGrid from "components/DatasetWarnings/MetricsDataGrid";
 import { ResponsivePlotWrapper, WarningPlot } from "components/PlotWrapper";
 import React from "react";
 import { DatasetWarningGroup } from "types/api";
+import Description from "components/Description";
+
+interface DescriptionMap {
+  [key: string]: string;
+}
+
+const WARNING_GROUP_DESCRIPTION: DescriptionMap = {
```

Comment:
I think you should use `Record<string, string>` instead of creating this vague `DescriptionMap`.
```suggestion
const WARNING_GROUP_DESCRIPTION: Record<string, string> = {
```